rerun_py/rerun_bindings/rerun_bindings.pyi

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Sequence
+
+import pyarrow as pa
+
+from .types import AnyColumn, ComponentLike, ViewContentsLike
+
+class ControlColumnDescriptor:
+    """A control-level column such as `RowId`."""
+
+class ControlColumnSelector:
+    """A selector for a control column."""
+
+    @staticmethod
+    def row_id() -> ControlColumnSelector: ...
+
+class TimeColumnDescriptor:
+    """A column containing the time values for when the component data was updated."""
+
+class TimeColumnSelector:
+    """A selector for a time column."""
+
+    def __init__(self, timeline: str): ...
```

Comment:
If we still plan to rename this and related types `IndexColumnSomethingSomething`, I guess now would be a better time than later?